gcc/rust/resolve/rust-forever-stack.hxx

```diff
@@ -97,6 +99,12 @@ ForeverStack<N>::pop ()
   update_cursor (cursor ().parent.value ());
 }
 
+static tl::expected<NodeId, DuplicateNameError>
+insert_inner (Rib &rib, std::string name, NodeId node, bool can_shadow)
+{
```

Comment:
I'm a bit confused by this. Is this function missing anything? Should'nt we keep the name somewhere ? If no, why keeping it in the parameters ?